### DISCUSSION TITLE
automatic Collection "Run All"

### DISCUSSION THREAD

**Discussion by JamesSalzer** [thunderclient/thunder-client-support#894] [Q&A] [answered]
Is there a way to automatically do a "Run All" of a collection when clicking the commit button in source control? Is there some kind of API with which I can use Thunder Client on script level?

**Comment by rangav** [comment, marked as answer]
We are planning to launch cli feature, which can solve this i think.

We have this feature planned for next update.
https://github.com/rangav/thunder-client-support/issues/764

Please add your use case in the above link.
